include/libtorrent/alert_manager.hpp

```diff
@@ -181,7 +188,7 @@ namespace libtorrent {
 		boost::function<void()> m_notify;
 
 		// the number of resume data alerts  in the alert queue
-		int m_num_queued_resume;
+		boost::atomic<int> m_num_queued_resume;
```

Comment:
I think you meant to change ``m_alert_mask``, not ``m_num_queued_resume``, right?

Reply:
or, I suppose both